### ISSUE TITLE
[Ulduar] General Vezax door does not open

### ISSUE THREAD

**Issue by Alorand** [dalaranwow/dalaran-wow#4755]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The door to general Vezax ulduar 10 man does not open after clearing all keepers This has happened in the past two weeks.

**Expected behaviour**: Tell us what should happen instead
The door should open immediately after clearing all keepers

**Steps to reproduce the problem**:

1.  Kill all of the keepers
2. The door doesn't open.
3. Wait for 1-2 hours for the doors to finally open. 

**Include proofs for this behaviour**
https://us.battle.net/forums/en/wow/topic/15098119834

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by urecheandu** [comment]
this can be worked around.

Kill all the keepers, with hodir as the last one.
Fight hodir normally, but, when he is about do die, wait for one more flash freeze, and do not free at least one of the frozen npcs. As vague as it is, this has worked so far, in my experience.

Should you fail to do this, the door will most likely bug, and you will have to wait for a server restart, for it to open.

**Comment by Rushor** [comment]
i think the reason for this is the combat bug that appears during the hodir encounter which was fixed here https://github.com/dalaranwow/dalaran-wow/issues/2087.

Nevertheless i'm going to add a special check for the encounters that should always open the gate once the keepers are killed.

**Comment by Rushor** [comment]
Scripts/Ulduar: Add CheckRequiredBosses for the Door to Vezax

* once all keepers are killed the door should be opened properly now
* closes: https://github.com/dalaranwow/dalaran-wow/issues/4755
* commit: 3708542424e8905bee932dd27d1278e6db7f2494
* commit: 1e9dd675e65c4a1c8316f20545fa918a0ae1a934

The issue was fixed by a developer and will be tested on the Public Test Realm in the future (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm in the future.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by JohnADC** [comment]
Bump, can you check this out, it's annoying to keep postponing the attunes and fragment collection cause it bugs almost every time. Thank you.

**Comment by datbunneh** [comment]
@Crispybuilt @Lothloryen This issue is still in place. It was even worse today - we created tickets and some GM actually opened it but it closed 2 seconds after that. GM was opening multiple times for us and it was closing couple seconds after every time.

**Comment by bradroid** [comment]
This bug occurred yesterday to our 10 man raid

**Comment by klaxce** [comment]
This bug has happened to us the last two weeks in a row, in Ulduar 10. We are unable to get the door to open. We have tried a number of "tricks", like going to each (dead) keeper's room, but haven't been successful so far. This week's Raid ID is 907.

**Comment by Lothloryen** [comment]
#5597
